priv/templates/new/lib/application_name/endpoint.ex

```diff
@@ -10,6 +10,7 @@ defmodule <%= application_module %>.Endpoint do
   # :code_reloader configuration of your endpoint.
   if code_reloading? do
     plug Phoenix.CodeReloader
+    plug Phoenix.LiveReloader
```

Comment:
Maybe this one should be first? We don't need to code reload code if we are hitting the LiveReloader router.